test-e2e/har/har.test.ts

```diff
@@ -0,0 +1,193 @@
+import 'mocha';
```

Comment:
We should include these file paths in our prettier config

Reply:
It should already be matching: https://github.com/segmentio/analytics.js-core/blob/8d34568433af0da9db7bd74ebb0a06f457fec22b/package.json#L16